src/renderer_gl.cpp

```diff
@@ -3432,13 +3615,13 @@ namespace bgfx { namespace gl
 			GL_CHECK(glColorMask(GL_TRUE, GL_TRUE, GL_TRUE, GL_TRUE) );
 
 			ProgramGL& program = m_program[_blitter.m_program.idx];
-			GL_CHECK(glUseProgram(program.m_id) );
-			GL_CHECK(glUniform1i(program.m_sampler[0], 0) );
+			GL_CHECK(GlUseProgram(program.m_id) );
```

Comment:
Move GL_CHECK to GL call callsite (idea of GL_CHECK is catch errors immediately, not later, and not from multiple GL calls).